workloads/kube-burner/run_clusterdensity_test_fromgit.sh

```diff
@@ -3,8 +3,8 @@
 set -e
 
 export WORKLOAD=cluster-density
-export METRICS_PROFILE=${METRICS_PROFILE:-metrics-aggregated.yaml}
-export TEST_JOB_ITERATIONS=${JOB_ITERATIONS:-1000}
```

Comment:
This line would still be needed I would think.

Reply:
https://github.com/cloud-bulldozer/e2e-benchmarking/pull/210#discussion_r698834308